Review user guide (#979)
ansible/receptor: docs/source/getting_started_guide/trying_sample_commands.rst

Try Sample Commands
###################

.. note::

>     The prior steps of network setup and receptor installation need to be completed in order for these command to work.

Reviewer: ```suggestion
    The previous network setup and receptor installation must be completed for these command to work.
```

>     The prior steps of network setup and receptor installation need to be completed in order for these command to work.

Reviewer 2: ```suggestion
    You must complete the prior steps of network setup and Receptor installation for these commands to work.
```
